etc/README.rst

```diff
@@ -0,0 +1,100 @@
+Availability Service Configuration
+==================================
+
+Placement
+---------
+
+Configuration file should be placed to */etc/oss/availability/config.json*.
+
+If there is an environment variable *AVAILABILITY_CONF* set, it is used as priority.
+In this case it is important to use absolute path to configuration file.
+
+Example:
+
+.. code-block::
+
+    export AVAILABILITY_CONF=/home/username/oss_configs/availability.json
+    python availability/main.py
+
+Configuration File Description
+------------------------------
+
+Configuration file is plain JSON document which defines
+`Flask <http://flask.pocoo.org>`_, connection to data storage and
+services quering configuration:
+
+Here is a simple example:
+
+.. code-block::
+
+  {
+      "flask": {
+          "PORT": 5020,
+          "HOST": "0.0.0.0",
+          "DEBUG": false
+      },
+      "elastic": "http://127.0.0.1:9200",
+      "period": 60,
+      "services": [
```

Comment:
We should try to avoid misspelling the names of regions. If you need to copy paste same name multple times there is a big chance to misspell it. 

Let's use different configuration format: 


```
"regions": [
      {
         "name": "west-1", 
         "services": [
               {"name": "nova", "url": "..."}, 
               {"name": "cinder", "url": "..."}, 
         ]
      }, 
      {
         "name": "west-2", 
         "services": [
               {"name": "keystone", "url": "..."}, 
               {"name": "cinder", "url": "..."}, 
         ]
      }
]
```